nights_watch/abstract_store.py

```diff
@@ -18,23 +18,28 @@
 Subclass for a specific storage engine.
 """
 
+from abc import ABCMeta, abstractmethod
+
 
 class AlreadyExistsError(Exception):
     pass
 
 
-class AbstractStore(object):
+class AbstractStore(object, metaclass=ABCMeta):
     def __init__(self, *args, **kwargs):
         """Args and behavior are dependent on the storage engine."""
-        raise NotImplementedError()
+        raise NotImplementedError('__init__')
 
+    @abstractmethod
```

Comment:
Given that methods marked @abstractmethod are never intended to actually be invoked, and subclasses won't even instantiate unless they're overridden, is there any point to having the `raise NotImplementedError` lines in them? Shouldn't they all just say `pass`?